packages/node/package.json

```diff
@@ -24,9 +24,8 @@
     "@nestjs/event-emitter": "^2.0.0",
     "@nestjs/platform-express": "^9.4.0",
     "@nestjs/schedule": "^3.0.1",
-    "@subql/common": "^3.5.1",
     "@subql/common-near": "workspace:*",
-    "@subql/node-core": "^10.1.1",
+    "@subql/node-core": "^10.10.0",
```

Comment:
This is quite the version jump, are we sure its compatible? I think there is the admin api stuff